src/fsharp/NicePrint.fs

```diff
@@ -1591,9 +1595,9 @@ module private TastDefinitionPrinting =
       let _,ty = generalizeTyconRef (mkLocalTyconRef tycon) 
       let start, name = 
           let n = tycon.DisplayName
-          if isClassTy g ty then (if simplified then None else Some "class" ), tagClass n
+          if isStructTy g ty then Some "struct", tagStruct n
```

Comment:
Does this word "struct" occur visibly anywhere?  If so could you screenshot an example please?  thanks!